tests/fixtures/config/sample-config.php

```diff
@@ -15,7 +15,7 @@
 
     'debug' => true,
 
-    "debugAgain"  =>  FALSE ,
+    "debugAgain"  =>  false ,
```

Comment:
```suggestion
    "debugAgain"  =>  false,
```
there's an extra space there.

Reply:
@LukeTowers I have a feeling that the spacing there is intentional, as it is here:
https://github.com/octobercms/library/blob/4099148a026b3d11091a76056811124db2c31aa7/tests/fixtures/config/sample-config.php#L54
I would imagine it's testing to see that the config values are still read correctly even when someone mangles the spacing.

Reply:
In that case we should leave it as FALSE and exclude this file from coding style checks @bennothommo

Reply:
Done :)